web/modules/custom/custom_services/custom_services.info.yml

```diff
@@ -0,0 +1,4 @@
+name: Custom Services
```

Comment:
That is not the Drupal way to define services. We should avoid creating modules that only contains services. These services should be defined in their respective module/s